app/src/main/java/com/codingblocks/cbonlineapp/dashboard/mycourses/DashboardMyCoursesFragment.kt

```diff
@@ -40,6 +42,8 @@ class DashboardMyCoursesFragment : Fragment() {
     private val viewModel by sharedViewModel<DashboardViewModel>()
     private val type = MutableLiveData<Int>()
     private val courseListAdapter = MyCourseListAdapter()
+    private val sharedPrefs by lazy { getSharedPrefs() }
+    private val COURSE_TYPE_POS = "course_filter_type"
```

Comment:
save all shared prefs in a constant file

Reply:
done